src/nvim/message.c

```diff
@@ -1450,6 +1450,9 @@ void msg_prt_line(char_u *s, int list)
       } else if (c == ' ' && trail != NULL && s > trail) {
         c = lcs_trail;
         attr = hl_attr(HLF_8);
+      } else if (c == ' ' && list && lcs_space != NUL) {
```

Comment:
I wanted to ask if you have any objections if I change `NUL` to `'\0'` in future pull requests? This seems to be conventional for good standard-compliant modern C (which is where we're going as far as I understand :smile: )